packages/lit-dev-tools-cjs/src/search/indexers/index-docs.ts

```diff
@@ -7,6 +7,9 @@ import path from 'path';
 import {docIndexer, UrlToFile, walkDir} from './utils.js';
 
 export const indexDocs = async (outputDir: string, idOffset = 0) => {
+  if (outputDir === '_dev') {
```

Comment:
I'm not sure this is needed, since indexDocs should not run in dev mode.

Reference: https://github.com/lit/lit.dev/blob/eece220ec85c61def9de6f420ba1a3e5d5d1d458/packages/lit-dev-content/.eleventy.js#L584-L588